master/buildbot/newsfragments/git-ssh-data-path-fix.bugfix

```diff
@@ -0,0 +1 @@
+:py:class:`~buildbot.util.git.GitStepMixin`: Change ssh data path to be more unique for a given builder
```

Comment:
I'd word this like: "Prevent builders from corrupting temporary ssh data by using `builldir` as part of the temporary path for that data". Right now it's not apparent what exact issue we're fixing.

Reply:
Yes I agree, I just wasn't sure this information was available here, but I will change it as you suggested.

Reply:
Yes, I see, I will change the wording to be more specific/explicit